Add js and py files for API pageswithprop

PRISCILAVILEMEN opened the pull request:
Created files that make requests to this API and printed the page titles as examples.

PRISCILAVILEMEN commented:
Hi @brendajerop,
can you help me reviewing my first pr? :)

PRISCILAVILEMEN commented:
> @PRISCILAVILEMEN Thank you for working on this!
> Some general comments:
> 
> * We need the PHP code as well, so please add it.
> * In your commit, make sure you add `API:Pageswithprop` to the readme files of Python, PHP and JavaScript folders.
> * Please provide a link to your sandbox page for this module.
> * We don't need to modify `.gitignore`.

@brendajerop , Thaks for the return..  

I want to know 
 1 * We need the PHP code as well, so please add it.
# I need to install something to run the code?

2- * In your commit, make sure you add `API:Pageswithprop` to the readme files of Python, PHP and JavaScript folders.
# about this task, I add the API, but the when click in the link, had error

3- * Please provide a link to your sandbox page for this module.
https://www.mediawiki.org/wiki/User:Prisicilavilemen/Sandbox/API:Pageswithprop

Is to Add this link?

PRISCILAVILEMEN commented:
@brendajerop ,  I need to know where I input this link : https://www.mediawiki.org/wiki/User:Prisicilavilemen/Sandbox/API:Pageswithprop , Can I help me please ;)

brendajerop commented:
> @brendajerop , I need to know where I input this link : https://www.mediawiki.org/wiki/User:Prisicilavilemen/Sandbox/API:Pageswithprop , Can you help me please ;)

In the pull request description or the comment section. You've already linked to it so that's fine :)

PRISCILAVILEMEN commented:
@brendajerop hi,
I read this info at contributing.md (located on the root directory)
It says: 

"For GET requests modules _only_, python, javascript and php sample code can be auto-generated by following the instructions below:"

As this file says **can be auto-generated** not **should be auto-generated**, so should I update this file to avoid further misleading interpretations?

PRISCILAVILEMEN commented:
I made the changes, please check, and give me feedback, thanks

brendajerop commented on php/get_pageswithprop.php:
```diff
@@ -0,0 +1,44 @@
+
+$output = curl_exec( $ch );
+
+if (curl_errno($ch)) { 
```
What does this `if statement` do? Do we need it?

brendajerop commented on php/get_pageswithprop.php:
```diff
@@ -0,0 +1,44 @@
+$ch = curl_init($url);
+
+curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );
+//curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
```
Remove this comment

PRISCILAVILEMEN commented:
@brendajerop , review my code ;) and give more feed backs

Best Regards

brendajerop commented:
@PRISCILAVILEMEN Please rebase your changes on top of those in master. To do this, run `git pull --rebase upstream master`.

brendajerop commented:
@PRISCILAVILEMEN What do you get when you run `git pull --rebase upstream master`?

PRISCILAVILEMEN commented:
@brendajerop , Sorry my rebase was not complete, but now I updated my branch and merged, please check. thanks

PRISCILAVILEMEN commented:
> @PRISCILAVILEMEN What do you get when you run `git pull --rebase upstream master?
My repo was a lot of commits behind, here is what I did before:
-  Rebased origin master from upstream master 
-  Rebased origin mw-pageswithprop from origin master

But it seems that all the changes weren't there.

I did rebase again on my branch directly from upstream master, and it seems that now it merged correctly.

This project is my first experience with git, but I think I am getting used to it.

Can you please give another look at the PR? 😄

brendajerop commented:
@PRISCILAVILEMEN I've left some feedback on the discussion page of your sandbox: https://www.mediawiki.org/wiki/User_talk:Prisicilavilemen/Sandbox/API:Pageswithprop :)
